pkg/labels/selector.go

```diff
@@ -72,10 +72,13 @@ const (
 	ExistsOperator       Operator = "exists"
 )
 
-//LabelSelector is a list of Requirements.
-type LabelSelector []Requirement
+func NewSelector() Selector {
```

Comment:
there's the `Add` func that to add `Requirements` but it could takes also a slice as input (and sort them internally).